indispensable-asn1/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -37,6 +30,13 @@ sealed class Asn1Element(
 
     }
 
+    /**
+     * The DER-encoded content bytes of this ASN.1 element.
+     * **Accessing this property on ASN.1 structures triggers encoding of the contained child nodes**, which may cause
+     * performance hits. Use with caution!
+     */
+    abstract val content: ByteArray
```

Comment:
per offline discussion: this should be a `ByteArrayView` that defers to `encodeToDer()`

Reply:
good thing you pointed this out. was a bad idea. if someone really need this for structures -> der encode and slice the last `contentLength` many bytes from the resulting byte array